src/cpp/include/openvino/genai/tokenizer.hpp

```diff
@@ -253,6 +254,8 @@ class OPENVINO_GENAI_EXPORTS Tokenizer {
     std::string get_eos_token() const;
     std::string get_pad_token() const;
 
+    Vocab get_vocab() const;
```

Comment:
Docstrings aren't our strong side, but you already added it for python, please add it here as well.

Reply:
Done